site/tests/support/test_contact_form.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# invenio-administration is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+"""Test support contact form."""
+
+import pytest
+from marshmallow import ValidationError
+
+from zenodo_rdm.support.support import ZenodoSupport
+
+
+@pytest.fixture
+def support_view():
+    return ZenodoSupport()
+
+
+@pytest.fixture
+def support_email_service(support_view):
+    return support_view.email_service
+
+
+@pytest.fixture
+def form_json_data():
+    return {
+        "name": "John Doe",
+        "email": "john_doe@invenio.org",
+        "category": "file-modification",
+        "description": "Hi! I need some help with my record ...",
+        "subject": "Record file is wrong",
+        "sysInfo": False,
+        "files": [],
+    }
+
+
+def test_endpoints():
```

Comment:
Should we clean unimplemented tests and make them an issue? I fear otherwise it will just be leftover code and we will forget about it

Reply:
I agree, they were there as placeholders but it's better to simply create issues

Reply:
issue: https://github.com/zenodo/zenodo-rdm/issues/87